### ISSUE TITLE
Units of orbital velocity in output file

### ISSUE THREAD

**Issue by ilyamandel** [TeamCOMPAS/COMPAS#314]
The column Orb_Velocity<SN in the BSE_Supernovae.csv file is labeled as having units of km/s, but in fact appears to contain the velocity in m/s.  If so, it should be recomputed (divided by 1000) before printing to contain the correct quantity.

**Comment by avigna** [comment]
From the code
`        case BINARY_PROPERTY::ORBITAL_VELOCITY_PRE_SUPERNOVA:                       value = OrbitalVelocityPreSN();                                             break;`
and from getter 
`    double              OrbitalVelocityPreSN() const                { return m_OrbitalVelocityPreSN; }`
so the variable of interest is 
`m_OrbitalVelocityPreSN`.

In previous commit 45d3d88dd0ec339f355d885d91925f21d62374dd
```
	// Calculate orbital velocity at some true anomaly psi - default is a circular orbit, V = sqrt(gm/a) = const.
    // Since this equation contains 'G', all other quantities must be in SI to get answer in ms^-1

	vK                       *= KM;                                                                                                 // convert vK to m s^-1.  Would be nice to draw this in nicer units to avoid this secion
	m_VRel                    = sqrt(G * (totalMass * MSOL_TO_KG) * ((2.0 / (m_Radius * AU)) - (1.0 / (m_SemiMajorAxis * AU))));    // orbital velocity
	m_uK                      = OPTIONS->UseFixedUK() ? OPTIONS->FixedUK() : vK / m_VRel;                                           // fix uK to user-defined value if required, otherwise calculate it.  uK is dimensionless
	m_OrbitalVelocityPreSN    = m_VRel;                                                                                             // since the kick velocity always occurs in equations as vk/vrel, we need to know vrel
```

In the most up-to-date commit d6565b405bfbc3d7df1e9b1c47e751229f0b6ce5 , which includes some of Reinhold's changes in supernovae:
```
        // Define gravitational constant G :=  km^3 Msol^-1 s^-2  - Defining G here as a preprocessing command improves readability without interferring with the existing G (in other units)
        #define G G_SN                                                 

        // Pre-SN parameters
        double a = m_SemiMajorAxisPrev*AU_TO_KM;                         // km  - Semi-Major axis
        double e = m_EccentricityPrev;                                   // --  - Eccentricity

        double m1 = m_Supernova->MassPrev();                             // Mo  - SN star pre-SN mass
        double m2 = m_Companion->MassPrev();                             // Mo  - CP star pre-SN mass
        double mb = m1 + m2;                                             // Mo  - Total binary pre-SN mass
        
        // Functions of Eccentric Anomaly
        m_Supernova->CalculateSNAnomalies(e);
        double cosEA = cos(m_Supernova->SN_EccentricAnomaly());        
        double sinEA = sin(m_Supernova->SN_EccentricAnomaly());

        double omega = sqrt(G*mb /(a*a*a));                              // orbits/s  - Keplerian orbital frequency

        Vector3d R = Vector3d( a*(cosEA-e),            
                               a*sqrt(1-e*e)*(sinEA),        
                               0.0                    );                 // km        - Relative position vector, from m1 to m2
        double   r = R.mag;                                              // km        - Separation

        Vector3d V = Vector3d( (-omega*a*a/r)*sinEA,   
                               (omega*a*a/r)*sqrt(1-e*e)*cosEA,  
                                0.0                            );        // km/s      - Relative velocity vector, in the m1 rest frame
        double   v = V.mag;                                              // km/s      - Relative orbital velocity, in the m1 rest frame
```

`m_OrbitalVelocityPreSN` seems to now be calculated in  [m_OrbitalVelocityPreSN] = km/s, as specified in the output file. 

Couldn't finish checking this because of a possible bug in preSupernova separation, but codewise it seems clean. Lets keep open for now and once we are happy with the values of preSN quantities, happy to calculate it by hand to double check.

**Comment by ilyamandel** [comment]
@reinhold-willcox -- this is implemented in the changes you are waiting on me to check, right?

**Comment by reinhold-willcox** [comment]
Sort of. It's implemented in the broader nskicks changes, but I'm submitting those incrementally now, so  it's on me to submit the next increment, not on you to check the previous PR.

**Comment by avigna** [comment]
Okay, let me know if I need to check this again in any PR.

**Comment by ilyamandel** [comment]
Fixed by @reinhold-willcox in PR #377 , closing.
